tests/fps_tests.py

```diff
@@ -10,6 +10,8 @@
 from FpsCheck import FpsCheck
 from check_sites import find_diff_sets
 
+WELL_KNOWN = "/.well-known/related-website-set.json"
+
```

Comment:
Can you rely on the constant directly?

```suggestion
from FpsCheck import FpsCheck, WELL_KNOWN
from check_sites import find_diff_sets

```

Reply:
Done